htdocs/modules/system/templates/admin/system_header.tpl

```diff
@@ -12,7 +12,7 @@
             <{/if}>
         <{/if}>
     <{/foreach}>
-    <{if $xo_sys_help|default:false}>
+    <{if isset($xo_sys_help)}>
```

Comment:
only for me in order to understand:
why replacing `<{if $xo_sys_help|default:false}>` by `<{if isset($xo_sys_help)}>`